rpmio/rpmio.c

```diff
@@ -1143,9 +1132,11 @@ static rpmzstd rpmzstdNew(int fdno, const char *fmode)
 	    goto err;
 	}
 
-	if (threads > 0)
+	threads = get_compression_threads (threads);
+	if (threads > 0) {
 	    if (ZSTD_isError (ZSTD_CCtx_setParameter(_stream, ZSTD_c_nbWorkers, threads)))
 		rpmlog(RPMLOG_WARNING, "zstd library does not support multi-threading\n");
```

Comment:
Not introduced in this commit of course, but while in the neighborhood... This should really either be RPMLOG_DEBUG or marked for translation, and I think RPMLOG_DEBUG is the way to go, just to avoid endless, useless warning spew.